examples/bundled/README.md

```diff
@@ -0,0 +1,36 @@
+# Bugsnag: Vue.js bundled example
+
+This example shows how you can use the Bugsnag JavaScript notifier with
+[Vue.js](https://vuejs.org/) that has a build step. Beyond the features shown in
+the [basic example](../basic), this setup shows how to initialize a `bugsnagClient` in a module
+that you can reuse in various parts of your application.
+
+This application was generated with [vue-cli](https://github.com/vuejs/vue-cli).
+
+## Setup
+
+Try this out with [your own Bugsnag account](https://app.bugsnag.com/user/new)!
+You'll be able to see how the errors are reported in the dashboard, how breadcrumbs
+are left, how errors are grouped and how they relate to the original source.
+
+To get set up, follow the instructions below. Don't forget to replace the placeholder
+API token with your own!
```

Comment:
Where does a user get their own API key?

Reply:
There is a link in the paragraphs above to create a new user. For existing users, there isn't a predictable link to get the api key because the url path contains the project slug. I guess we could be verbose here and give instructions, but I think that might be overkill?